test/graphql/types/userList.spec.ts

```
@@ -0,0 +1,113 @@
+	it("should throw an error when first is less than 1", async () => {
+		const result = await executeOperation({ first: 0 });
+		expect(result.errors).toBeDefined();
+		expect(result.errors?.[0]?.message).toContain("Invalid arguments");
+		expect(mockContext.drizzleClient.query.usersTable.findMany).not.toHaveBeenCalled();
+	});
+
+	it("should return unauthorized error if user is not authenticated", async () => {
+		const testContext = { ...mockContext, currentClient: { isAuthenticated: false, user: null } };
+		const result = await execute({
+			schema,
+			document: parse(`query { userList { id name email } }`),
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)